doc/model/train-energy.md

```diff
@@ -0,0 +1,60 @@
+# Fit energy
+
```

Comment:
Add position of the example: `example/water/se_e2_a/input.json`, where reader can find the sections "fitting" and "loss"

Reply:
Resolved.